src/extension.ts

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 import * as vscode from 'vscode';
+import * as p from 'path';
```

Comment:
`p` looks too simple to me, it's not easy to understand 😂

Reply:
check the latest commit

Reply:
awesome, really appreciate it.